war/src/main/js/components/tooltips/index.js

```diff
@@ -21,6 +20,10 @@ function registerTooltip(element) {
 
   const tooltip = element.getAttribute("tooltip");
   const htmlTooltip = element.getAttribute("data-html-tooltip");
+  let appendTo = document.body;
+  if (element.hasAttribute("data-tooltip-append-to")) {
+    appendTo = element.getAttribute("data-tooltip-append-to");
```

Comment:
Should we maybe force here the value to `parent`.  If someone passes anything other than `parent` this would not be a valid value. Or directly treat it as an ID if other than parent and look up this id and pass it on.

Reply:
Probably console.warn and force to parent

Reply:
Made it explicit in the attribute name to append to parent